.github/workflows/ci.yml

```diff
@@ -15,21 +15,46 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
-        django: ['3.2', '4.0', '4.1', '4.2', 'main']
+        python-version:
+        - '3.7'
+        - '3.8'
+        - '3.9'
+        - '3.10'
+        - '3.11'
+        django:
+        - '3.2'
+        - '4.0'
+        - '4.1'
+        - '4.2'
+        - '5.0'
+        - 'main'
         exclude:
           - python-version: '3.7'
             django: '4.0'
```

Comment:
I suggest a follow-up PR to remove 3.7, since it’s EOL. Sound good?

Reply:
Absolutely—I'll take care of that, thank you!